scripts/valhalla_build_config

```diff
@@ -30,7 +31,7 @@ config = {
         'user_agent': Optional(str),
         'tile_url': Optional(str),
         'tile_url_gz': Optional(bool),
-        'concurrency': Optional(int),
+        'concurrency': mp.cpu_count(),
```

Comment:
i assume the main benefit of doing this is to demystify what the code is doing when you leave this out, right? i cant argue with that!